README.md

```diff
@@ -59,7 +59,12 @@ See [client_example_test.go](client_example_test.go)
 - Data that is sent in the HTTP body is passed as struct
   pointer to API methods.
 - Pointers instead of values are used to represent fields in body messages
-  structs. This allows to represent unset fields correctly.
+  structs. \
+  The bunny.net API does not define which values are assumed if a field
+  is omitted in a request. 
+  Using pointers allows to distinguish between unset and empty fields. This
+  prevents discrepancy between the interpretation of missing fields of the
+  bunny.net API and bunny-go.
```

Comment:
Maybe write that without pointers is not possible to differentiate between a default value and an omitted value, and the API design differentiate them.

Reply:
I changed it, please have a look again

Reply:
Perfect.